### ISSUE TITLE
SketchyBar Interrupts ScreenCapture Permissions

### ISSUE THREAD

**Issue by cpt-kernel** [FelixKratz/SketchyBar#518]
In Sonoma 14.4 when SketchyBar is on via 'brew services start sketchybar' I am unable to use applications that require Screen Recording permissions.

Example, MonoSnap has the screen recording toggled ON via: 
System Preferences > Security and Privacy > Screen Recording.

But when SketchyBar is enabled, using the program prompts to provide permission which already exists. 
This issue is easily reproducible. 

This may be related to the way SketchyBar is acting as an overlay for menubar items.
<img width="1300" alt="Screenshot 2024-03-14 at 12 57 32 PM" src="https://github.com/FelixKratz/SketchyBar/assets/23063172/142cacfc-dac7-4276-86b0-748979d41cce">

I am continuing to investigate, but would appreciate any help with this matter.

**Comment by FelixKratz** [comment]
Can you reproduce this with another tool that requires those permission?

**Comment by cpt-kernel** [comment]
This was happening with TextSniper, but that went away after I added SketchyBar to the screen recording permissions.
So now the problem appears to be isolated to Monosnap's screen clipping, all other functions of the program work.

I have emailed their support as well, but they're pretty unsure of what this issue could be.

I'm willing to provide logs or anything that may help.
If this issue is truly isolated to one program, one person… well, I'll understand it not being a priority. 
Happy to throw some coffee money your way. 

MonoSnap has been an important part of my workflow, and after finding SketchyBar I'm really liking it.
